css/style.css

```diff
@@ -1 +1,1087 @@
-body {}
\ No newline at end of file
+:root {
+  --btn-primary-color: rgba(33, 231, 134, 1);
+  --secondary-color: rgba(20, 27, 34, 1);
+  --article-bg-color: rgba(255, 255, 255, 0.05);
+  --main-background-color: rgba(13, 13, 17, 1);
+  --text-color: rgba(194, 195, 197, 1);
+  --inactive-color: rgba(136, 139, 142, 1);
+  --text-font: "Poppins", sans-serif;
```

Comment:
Provide fallback fonts and family types at the end of the font list in custom properties too.  
For example: font-family: "Roboto", Arial, sans-serif;

Reply:
#fixed